libtrap/src/ifc_tls.c

```diff
@@ -133,12 +145,81 @@ static SSL_CTX *tlsclient_create_context()
    return ctx;
 }
 
-static int tlsserver_configure_context(SSL_CTX *ctx, const char *key, const char *crt)
+/**
+ * \brief Configure context of ssl server.
+ * \param[in] arg  pointer to SSL 
```

Comment:
It can be better for reader to refer to struct where `SSL_CTX *` and `SSL *` pointers are stored... In doxygen, it is easy to create links between different doc places.